pkg/config/agentfeaturesconfig.go

```diff
@@ -0,0 +1,76 @@
+package config
+
+import (
+	"github.com/Axway/agent-sdk/pkg/cmd/properties"
+)
+
+// AgentFeaturesConfig - Interface to get agent features Config
+type AgentFeaturesConfig interface {
+	ConnectionToCentralEnabled() bool
+	ProcessSystemSignalsEnabled() bool
+	AgentVersionCheckerEnabled() bool
```

Comment:
May be we can remove the version checker config in CentralConfig

Reply:
Agreed it should go. Was afraid I'd break something, but suspect the Amplify Gateway is the only user of this flag. Will I just remove it ?